README.markdown

```diff
@@ -1,4 +1,4 @@
-Symfony Documentation
+Symfony Documentation [![Build Status](https://travis-ci.org/symfony/symfony-docs.svg?branch=master)](https://travis-ci.org/symfony/symfony-docs)
```

Comment:
Not sure if actually needed. The travis build is here just so people are sure their docs won't break the build. Don't see a value in showing that badge in the readme, maybe I am wrong.

Reply:
Recently the master build was broken, and it was not visible on the main page.
So the build of my PR it was broken too, and I spent some time understanding that it's not me :)